gradle-sls-packaging/src/main/resources/init.sh

```diff
@@ -42,15 +42,22 @@ STATIC_LAUNCHER_CHECK_CONFIG="service/bin/launcher-check.yml"
 
 DEPRECATION_MESSAGE="Command is deprecated: the next major release of sls-packaging will only support start/status/stop"
 
+function print_help() {
+    echo "$0 forwards commands to go-init with the following usage."
+    echo ""
+    exec $GO_INIT_CMD "$@"
+}
+
+if [[ "$ACTION" =~ ^(-h|--help)$ && "$2" =~ ^(|start|stop|status)$ ]]; then
+    print_help --help "$2"
+fi
+
 case $ACTION in
-start)
-    $GO_INIT_CMD start
-;;
-status)
-    $GO_INIT_CMD status
-;;
-stop)
-    $GO_INIT_CMD stop
+start|status|stop)
+    if [[ "$2" == "--help" ]]; then
```

Comment:
why do we only handle `--help` here, but `--help|-h` above?

Reply:
Fixed.